app/routes/task_routes.py

```diff
@@ -1 +1,99 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, jsonify
+from ..db import db
+from app.models.task import Task
+from .route_utilities import validate_model, create_model
+from datetime import datetime
+import requests, os
+
+
+bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+    task_data, status_code = create_model(Task, request_body)
+    return jsonify({"task": task_data}), status_code
+
+
+@bp.get("")
+def get_all_tasks():
+    query = db.select(Task)
+
+    title_param = request.args.get("title")
+    if title_param:
+        query = query.where(Task.title.ilike(f"%{title_param}%"))
+
+    description_param = request.args.get("description")
+    if description_param:
+        query = query.where(Task.description.ilike(f"%{description_param}%"))
+
+    sort_param = request.args.get("sort")
+    if sort_param == "desc":
+        query = query.order_by(Task.title.desc())
+    else:
+        query = query.order_by(Task.title.asc())
+
+    tasks = db.session.scalars(query)
+
+    return [task.to_dict() for task in tasks]
```

Comment:
Nice job directly returning a list and **not** using jsonify. The newer version of Flask we use, under the hood, is able to convert a list or a dictionary into JSON without needing us to call `jsonify` on either the list or dictionary.